layouts/_default/baseof.html

```diff
@@ -41,6 +41,7 @@ <h1 class="header__site-name">
 <p>{{ with $.Param "copyright" }}{{ . | markdownify }}{{ else }}<span>© {{ site.Title }}</span>{{ end }}</p>
 {{ with $.Param "footer" }}<p>{{ . | markdownify }}</p>{{ end }}
 {{ if $.Param "poweredby" }}<p>Powered by <a href="https://gohugo.io/">Hugo</a> and the <a href="https://github.com/frjo/hugo-theme-zen">Zen theme</a>.</p>{{ end }}
+{{ if or (site.Params.math) (.Params.math) }}{{ partial "math.html" . }}{{ end -}}
```

Comment:
Write it like this instead: `{{ if $.Param "math" }}`

This will first check for page params and then for site params. Neat Hugo shortcut.

Reply:
With the other changes we can move this up to the header.